azerothcore/azerothcore-wotlk
# fix(DB/GO): Chests for "Grimtotem Spying" not despawning

## Changes Proposed:
- Decreases the respawn time of the three chests needed for the quest "Grimtotem Spying" to 0 s, so that they don't despawn. 
It is only possible to interact with the chests if you have the quest, and you can't pick up more than one of the Secret Notes from the chest.

## Issues Addressed:

-  Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7914
-  Closes https://github.com/chromiecraft/chromiecraft/issues/1764

## SOURCE:

https://youtu.be/TArHG3QL9CI?t=237 - 3:57
https://youtu.be/9atTQbckRXY?t=69 - 1:09


## Tests Performed:

- Ran SQL, no errors
- checked db and in-game

## How to Test the Changes:


1. .q add 5064
2. .go o 17199
3. .go o 17201
4. .go o 17200
5. Gather the Secret Notes and observe, that the chests are not despawning

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
